### ISSUE TITLE
Alias from two different files is not displayed

### ISSUE THREAD

**Issue by bendobos** [UX-and-I/storybook-design-token#10]
Hi there! I noticed the following behavior:

Have two different css files, for example:
```css
/* file1.css */

/**
 * @tokens Colors 1
 * @presenter Color
 */
 --gray-300: #DDDDDD;
 --gray-400: #AAAAAA;
 --gray-500: #777777;
```

```css
/**
 * @tokens Colors 2
 * @presenter Color
 */

/* file2.css */
--gray-default: var(--gray-300);

--another-var: #BADA55;
```

In Storybook, `--gray-default` is not shown at all, while `--another-var` is shown. `--gray-default` is also not displayed as an alias.
When I move everything in one file, the aliases are detected as expected.
Having the same value for `@tokens` in both files shows the same behavior.

**Comment by Sqrrl** [comment]
Thanks. Will look into that.

**Comment by bendobos** [comment]
Fixed a typo in my example :D.

**Comment by Sqrrl** [comment]
Fixed with v0.7.1
